src/content/docs/guides/getting-started.md

```diff
@@ -13,204 +11,300 @@ All of the data collected by the HTTP Archive is available via [Google BigQuery]
+Next let's summarize all of the HTTP requests by a type, and the number of pages that contain at least one request of that type. In the example below, you can see that I added `type` to the SELECT clause, added a GROUP clause and sorted the results by types that have the most requests.
 
 ```sql
 SELECT
-  mimeType,
-  COUNT(distinct pageid) total_pages,
+  type,
+  COUNT(DISTINCT page) total_pages,
   COUNT(0) total_requests
 FROM
-  `httparchive.summary_requests.2018_09_01_desktop`requests
+  `httparchive.all.requests`
+WHERE
+  date = "2024-06-01"
+  AND client = "desktop"
+  AND is_root_page
 GROUP BY
-  mimeType
+  type
 ORDER BY
   total_requests DESC
 ```
 
 Now things are starting to get interesting.
 
-![Simple JOIN Example](https://github.com/HTTPArchive/httparchive.org/raw/main/docs/images/mimeType_summary_example_query.jpg)
+![Simple JOIN Example](./type_summary_example_query.png)
 
-So let's try to learn something from this basic example.   We know from the first example that there are 1.2 million URLs in the latest HTTP Archive dataset. Let's calculate the percent of pages that have each mimeType. To do this, we'll divide the number of pages by the total pages (using our first query as a subquery). Then we'll use a `ROUND()` function to trim the result to 2 decimal points.
+So let's try to learn something from this basic example. We know from the first example that there are 12.7 million URLs in the latest HTTP Archive dataset. Let's calculate the percent of pages that have each mimeType. To do this, we'll divide the number of pages by the total pages (using our first query as a subquery). Then we'll use a `ROUND()` function to trim the result to 2 decimal points.
 
 ```sql
+WITH requests AS (
+  SELECT
+    type,
+    page,
+    COUNT(0) OVER () AS requests_total,
+    COUNT(DISTINCT page) OVER () AS pages_total,
+  FROM `httparchive.all.requests`
+  WHERE
+    date = "2024-06-01"
+    AND client = "desktop"
+    AND is_root_page
+)
+
 SELECT
-  mimeType,
-  COUNT(distinct pageid) total_pages,
-  COUNT(0) total_requests,
-  ROUND(
-    COUNT(distinct pageid) / (
-      SELECT
-        COUNT(0)
-      FROM
-        `httparchive.summary_pages.2018_09_01_desktop`
-    )
-    , 2) percent_pages
-FROM
-  `httparchive.summary_requests.2018_09_01_desktop`requests
+  type,
+  COUNT(DISTINCT page) AS pages_total,
+  ANY_VALUE(requests_total) AS requests_total,
+  ROUND(COUNT(DISTINCT page) / ANY_VALUE(pages_total), 2) AS pages_percent
+FROM requests
 GROUP BY
-  mimeType
+  type
 ORDER BY
-  total_requests DESC
+  requests_total DESC
 ```
 
 When analyzing the results from this, you can see the % of websites that use different Content-Types for their JavaScript, you can see that 93% of sites have at least one PNG image, 89% have at least 1 GIF, 48% use JSON, and 3% of sites have MP4 videos on their homepage, etc.
```

Comment:
Needs an update. Instead of the MIME type, we're looking at resource types (script, css, image, etc).